pages/collection/[chain]/[contract].tsx

```diff
@@ -97,6 +97,7 @@ const CollectionPage: NextPage<Props> = ({ id, ssr }) => {
     collection: id,
     sortBy: 'floorAskPrice',
     sortDirection: 'asc',
```

Comment:
This also needs to be added to the SSR api request

Reply:
Resolved!